projects/plugins/jetpack/extensions/plugins/publicize/components/panel/index.js

```diff
@@ -91,6 +94,7 @@ const PublicizePanel = ( { prePublish } ) => {
 						isPublicizeEnabled={ isPublicizeEnabled }
 						isRePublicizeFeatureEnabled={ isRePublicizeFeatureEnabled }
 						isPublicizeDisabledBySitePlan={ isPublicizeDisabledBySitePlan }
+						connectionsAdminUrl={ connectionsAdminUrl }
```

Comment:
@pablinos @danielpost  Should we be taking the user may be to Jetpack Cloud here instead of https://wordpress.com/marketing/connections/<url> or is this ok to do?

Reply:
Yes, I think we have a connections URL we can use in the Jetpack redirects. It should already be available as we use it for the ['Connect an account' link](https://github.com/Automattic/jetpack/blob/2447e267e40f2d133a5c15c2320b776e4c395ec8/projects/js-packages/publicize-components/src/components/settings-button/index.js#L36)

Reply:
I tried this, but the getJetpackData()?.publicizeConnectionsUrl is not available here 🤔

Reply:
Good idea! Now I'll use the one we use with the ['Connect an account' link](https://github.com/Automattic/jetpack/blob/2447e267e40f2d133a5c15c2320b776e4c395ec8/projects/js-packages/publicize-components/src/components/settings-button/index.js#L36) then.